addons/website/static/src/js/website.js

```diff
@@ -1,7 +1,7 @@
 odoo.define('website.website', function (require) {
 'use strict';
 
-var weContext = require('web_editor.context');
```

Comment:
Actually website.website is not needed anymore and should be included directly in website.context as this was only in charge of extending the context stuff. I can do it if you want, but either remove all the context-related stuff in this PR or do all the needed changes :wink: Don't want this to be included in the same commit as web_editor.base.